.github/workflows/testing-integration.yml

```diff
@@ -20,7 +20,8 @@ jobs:
       matrix:
         php: [ '8.0', '7.4' ]
         wordpress: [ '6.1', '5.9' ]
-        acf_pro: [ true, false ]
+        acf_plugin_slug: [ 'advanced-custom-fields', 'advanced-custom-fields-pro' ]
+        acf_extended_plugin_slug: [ 'acf-extended', 'acf-extended-pro' ]
```

Comment:
@justlevine so I think this will add a LOT more unnecessary tests to the matrix. 

ACF Extended and ACF Extended PRO **_both_** require ACF PRO to be active, so we don't need to add test matrix for ACF FREE + ACF EXTENDED or ACF FREE + ACF EXTENDED PRO. . .this will add a lot of tests to the matrix that are unnecessary. 

This is why I was doing things with the env vars

Reply:
Thanks for letting me know - I've never used ACF Extended, so I was basing it off the logic in `app.setup.sh`.

https://github.com/wp-graphql/wpgraphql-acf/commit/edf9f63947539221a96f97d581a9b393301185e6 moves `acf_extended_plugin_slug` from the matrix to a set env var, which should generate the same # of tests as before (it seems you're _only_ ever testing against extended pro and the CI just skips scenarios ).